proteuslib/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -234,7 +234,7 @@ def calculate_operating_pressure(feed_state_block=None, over_pressure=0.15,
     return value(t.brine[0].pressure_osm) * (1 + over_pressure)
 
 
-def calculate_RO_area(unit=None, water_recovery=0.5, solver=None):
+def calculate_RO_area(unit=None, water_recovery=0.5, solver_dict=None):
```

Comment:
I would prefer if we maintained the `solver` keyword argument and add a `optarg` keyword argument. The `solver` could be a string which we use to create a solver object within this function using the options. This maintains consistency with IDAES.

I'm happy to make this change myself.

Reply:
I'll make an issue for it. There's a couple of other things I could add.

Reply:
Thanks! Part of #102.